app/controllers/api/api_controller.rb

```diff
@@ -0,0 +1,23 @@
+class Api::ApiController < Api::V1::BaseApiController
+  def index
+    json = Rails.cache.fetch('API_JSON_RESPONSE', expires_in: 1.day) do
```

Comment:
Is caching necessary, and is it cleared after a production deploy with a version update?